rx-android-extensions/src/main/java/com/appunite/rx/android/widget/RxFragmentManagerMore.java

```diff
@@ -0,0 +1,94 @@
+package com.appunite.rx.android.widget;
+
+import android.support.annotation.IdRes;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+
+import javax.annotation.Nonnull;
+
+import rx.functions.Action1;
+
+public class RxFragmentManagerMore {
+
+    @Nonnull
+    public static Action1<? super Object> popBackStack(@Nonnull final FragmentManager fragmentManager) {
+        return new Action1<Object>() {
+            @Override
+            public void call(Object ignored) {
+                fragmentManager.popBackStack();
+            }
+        };
+    }
+
+    @Nonnull
+    public static Action1<? super Object> add(@Nonnull final FragmentManager fragmentManager,
```

Comment:
I'm not sure its usefull have this implemented because you pre-create fragment instance before it's needed and you can acctidently add the same instance in two places